src/Components/OurClient/OurClients.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from "react";
+import React from "react";
 import IBM from "../../Assets/ibm.png";
```

Comment:
@Ashutosh-Maurya-87 Please do import in groups
1. React
2. MUI
3. Others
4. Style files
5. components
6. helper files
7. configs
8. utils
9.  other